doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -168,6 +168,20 @@ Additional scenario XML elements:
   Specify whether force to disable software workaround for Machine Check
   Error on Page Size Change is enabled.
 
+``IVSHMEM`` (a child node of ``FEATURE``):
+  Specify the IVSHMEM configuration
+
+``IVSHMEM_ENABLED`` (a child node of ``FEATURE/IVSHMEM``):
+  Specify whether Enable Share Memory between VMs by IVSHMEM.
+
+``IVSHMEM_REGION`` (a child node of ``FEATURE/IVSHMEM``):
+  Specify the name, size, IDs of communication VMs of share memory region,
+  input format: hv:dm/<;shm name>;,<;shm size in MB>;,<;vm id:vm id[:vm id,...]>.
```

Comment:
This is very hard to read... any suggestion on how we can make this more readable @dbkinder ?

Reply:
my submitted edit recommendation is to use words rather than a syntax description using notation that's unfamiliar.

Reply:
> my submitted edit recommendation is to use words rather than a syntax description using notation that's unfamiliar.

yes, I have updated it as yours, the reply  is just to explain to you, it won't be added in doc.